conftest.py

```diff
@@ -1,8 +1,26 @@
 # project root conftest.py
+import fauxfactory
 
 pytest_plugins = [
     "mta.fixtures.apis",
     "mta.fixtures.application_inventory",
     "mta.fixtures.credentials",
     "mta.fixtures.tags",
+    "mta.fixtures.stakeholder",
+    "mta.fixtures.business_service",
 ]
+
+
+def generate_name(length=20, start=None, separator="-"):
+    """Generate a random name"""
+    return fauxfactory.gen_alphanumeric(length=length, start=start, separator=separator)
+
+
+def generate_email():
+    """Generate a random email"""
+    return fauxfactory.gen_email()
+
+
+def generate_description(length=120, start=None):
+    """Generate a random description"""
+    return fauxfactory.gen_alphanumeric(length=length, start=start)
```

Comment:
@kpunwatk I see this function as redundant. you can use `generate_name` function above and override the default length to 120.
Please rename `generate_name` to `generate_alphanumeric` or `generate_string`

Reply:
Thanks @mguetta1 for the suggestion. Done renamed to `generate_string`